src/contextProviders/DonationProvider.jsx

```diff
@@ -56,6 +56,7 @@ class DonationProvider extends Component {
             query: {
               giverAddress: this.props.currentUser.address,
               amountRemaining: { $ne: 0 },
+              pendingAmountRemaining: { $ne: 0 },
```

Comment:
why was this added? Doesn't this do an AND query

Reply:
I have a question here. This amount is the new `amountRemaining` once confirmed, right? In that case IMHO this is fine because both of these values need to be greater than 0, right? If anything I should actually be using the `pendingAmountRemaining` all the time rather than `amountRemaining`.